Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/SMGs/smgs.yml

```
@@ -0,0 +1,51 @@
+# Rubber
+- type: entity
+  name: Drozd
+  parent: WeaponSubMachineGunDrozd
+  id: WeaponSubMachineGunDrozdRubber
+  suffix: Non-Lethal
+  components:
+    - type: ItemSlots
+      slots:
+        gun_magazine:
+          name: Magazine
+          startingItem: MagazinePistolSubMachineGunRubber
+          insertSound: /Audio/Weapons/Guns/MagIn/smg_magin.ogg
+          ejectSound: /Audio/Weapons/Guns/MagOut/smg_magout.ogg
+          priority: 2
+          whitelist:
+            tags:
+              - MagazinePistolSubMachineGun
+        gun_chamber:
+          name: Chamber
+          startingItem: CartridgePistolRubber
+          priority: 1
+          whitelist:
+            tags:
+              - CartridgePistol
+
+- type: entity
+  name: Vector
+  parent: WeaponSubMachineGunVector
+  id: WeaponSubMachineGunVectorRubber
+  description: An excellent fully automatic Heavy SMG. Uses .45 magnum ammo.
+  suffix: Non-Lethal
```

If the vector is deprecated, the non-lethal version should say so as well?
```suggestion
  suffix: Deprecated, Non-Lethal
```

---

I might as well remove it completely